lms/static/sass/course/ccx_coach/_dashboard.scss

```diff
@@ -18,6 +18,11 @@ table.ccx-schedule {
     th, td {
```

Comment:
Please make sure this file's contents adhere to our Sass/Styling Styleguide - https://github.com/edx/ux-pattern-library/wiki/Styleguide:-Sass-&-CSS.